specification/vi/resource-manager/Microsoft.VideoIndexer/preview/2024-06-01-preview/vi.json

```diff
@@ -0,0 +1,1427 @@
+          "description": "The user assigned identity to be used to grant permissions",
+          "example": "/subscriptions/xxx/resourceGroups/contoso-videoAnalyzer-rg/providers/Microsoft.ManagedIdentity/userAssignedIdentities/contoso-videoAnalyzer-mi"
+        }
+      }
+    },
+    "AccessToken": {
+      "description": "Azure Video Indexer access token.",
+      "type": "object",
+      "properties": {
+        "accessToken": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The access token.",
+          "x-ms-secret": true,
+          "example": "eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJPbmxpbmUgSldUIEJ1aWxkZXIiLCJpYXQiOjE2MjI0NDc3MzMsImV4cCI6MTY1Mzk4MzczMywiYXVkIjoid3d3LmV4YW1wbGUuY29tIiwic3ViIjoianJvY2tldEBleGFtcGxlLmNvbSIsIkdpdmVuTmFtZSI6IkpvaG5ueSIsIlN1cm5hbWUiOiJSb2NrZXQiLCJFbWFpbCI6Impyb2NrZXRAZXhhbXBsZS5jb20iLCJSb2xlIjpbIk1hbmFnZXIiLCJQcm9qZWN0IEFkbWluaXN0cmF0b3IiXX0"
+        }
+      }
+    },
+    "Operation": {
+      "description": "Operation detail payload",
+      "type": "object",
+      "properties": {
+        "name": {
+          "description": "Name of the operation",
+          "readOnly": true,
+          "type": "string",
+          "example": "Microsoft.VideoIndexer/accounts/read"
+        },
+        "isDataAction": {
+          "description": "Indicates whether the operation is a data action",
+          "readOnly": true,
+          "type": "boolean",
+          "example": false
+        },
+        "actionType": {
+          "description": "Indicates the action type.",
+          "readOnly": true,
+          "type": "string",
+          "example": "Internal"
+        },
+        "display": {
+          "$ref": "#/definitions/OperationDisplay",
+          "readOnly": true,
+          "description": "Display of the operation"
+        },
+        "origin": {
+          "description": "Origin of the operation",
+          "readOnly": true,
+          "type": "string",
+          "example": "user"
+        }
+      }
+    },
+    "OperationDisplay": {
+      "description": "Operation display payload",
+      "type": "object",
+      "properties": {
+        "provider": {
+          "description": "Resource provider of the operation",
+          "readOnly": true,
+          "type": "string",
+          "example": "Microsoft Azure Video Indexer"
+        },
+        "resource": {
+          "description": "Resource of the operation",
+          "readOnly": true,
+          "type": "string",
+          "example": "accounts"
+        },
+        "operation": {
+          "description": "Localized friendly name for the operation",
+          "readOnly": true,
+          "type": "string",
+          "example": "Read accounts"
+        },
+        "description": {
+          "description": "Localized friendly description for the operation",
+          "readOnly": true,
+          "type": "string",
+          "example": "Read accounts details"
+        }
+      }
+    },
```

Comment:
Are these redundant and removable now you removed the local definition of operationListResult?

Reply:
@TimLovellSmith  absolutely ! great catch. all got removed and also moved to using common-types/ErrorDetails